README.md

```diff
@@ -43,44 +42,57 @@ How about:
 ![Animated SVG of the tldr client displaying the tar command.](images/tldr.svg)
 
 This repository is just that: an ever-growing collection of examples
-for the most common UNIX, Linux, macOS, SunOS, and Windows command-line tools.
+
+for the most common UNIX, Linux, macOS, SunOS, Android and Windows command-line tools.
 
 ## How do I use it?
 
 A popular and convenient way to access these pages on your computer
 is to install the [Node.js client](https://github.com/tldr-pages/tldr-node-client),
 which is supported by the tldr-pages project maintainers:
 
-    npm install -g tldr
+```sh
+npm install -g tldr
+```
+
+Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
```

Comment:
```suggestion
Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip`.
```
Isn't `pip` an alias to `pip3` on most machines? And even if it isn't, wouldn't `tldr` still be installable with older versions?

Reply:
not 100% on this but machines with python2, I think  `pip` refers to python2.
I can't tell if I am correct, please correct me if I am not.

Reply:
But would machines with Python2 even be able to run pip3, which needs Python3?

Well, I think we could just keep pip3, since it doesn't really matter.